c2corg_ui/static/js/editing/documentediting.js

```diff
@@ -666,7 +666,7 @@ app.ConfirmSaveController = function($uibModalInstance, appDocument, appLang) {
    * @type {string}
    * @export
    */
-  this.quality = appDocument.document.quality;
+  this.quality = appDocument.document.quality || 'draft';
```

Comment:
A small downside of having an hard-coded default quality will prevent to have it updated if the list of qualities changes in https://github.com/c2corg/v6_common/blob/master/c2corg_common/attributes.py#L601-L607

A cleaner solution would have been to retrieve this value from the config file above. But I admit it would have been quite awkward because it would probably imply to inject this value through a module constant passed in the mako template as in https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/base.html#L137-L151

It's probably unrequired complexity, then OK for me.